docs/lib/getting-started/fragments/android/integrate.md

```diff
@@ -1,155 +1,459 @@
+
+1. Open `MainActivity` and add the following code to the bottom of the `onCreate()` method:
+
+  <amplify-block-switcher>
+  <amplify-block name="Java">
+  
+  ```java
+  try {
+      Amplify.addPlugin(new AWSDataStorePlugin());
+      Amplify.configure(getApplicationContext());
+
+      Log.i("Tutorial", "Initialized Amplify");
+  } catch (AmplifyException e) {
+      Log.e("Tutorial", "Could not initialize Amplify", e);
+  }
+  ```
+
+  </amplify-block>
+
+  <amplify-block name="Kotlin">
+
+  ```kotlin
+  try {
+      Amplify.addPlugin(new AWSDataStorePlugin())
+      Amplify.configure(applicationContext)
+
+      Log.i("Tutorial", "Initialized Amplify")
+  } catch (e: AmplifyException) {
+      Log.e("Tutorial", "Could not initialize Amplify", e)
+  }
+  ```
+
+  </amplify-block>
+  </amplify-block-switcher>
+
+1. Run the application. In logcat, you'll see a log line indicating success:
+
+    ```console
+    com.example.todo I/Tutorial: Initialized Amplify
+    ```
+
+    To make this easier to find, enter the string "Tutorial" into the search field.
+
+## Create a Todo
+
+Next, you'll create a Todo and save it to DataStore.
+
+1. Open `MainActivity` and add the following code to the bottom of the `onCreate()` method:
+
+  <amplify-block-switcher>
+  <amplify-block name="Java">
+
+  ```java
+  Todo item = Todo.builder()
+          .name("Build Android application")
+          .description("Build an Android Application using Amplify")
+          .build();
+  ```
+
+  </amplify-block>
+
+  <amplify-block name="Kotlin">
+
+  ```kotlin
+  val item: Todo = Todo.builder()
+        .name("Build Android application")
+        .description("Build an Android Application using Amplify")
+        .build()
+  ```
+
+  </amplify-block>
+  </amplify-block-switcher>
+
+  This code creates a Todo item with two properties: a title and a description. This is a plain object that isn't stored in DataStore yet.
+
+1. Below that, add the code to save the item to DataStore:
+
+  <amplify-block-switcher>
+  <amplify-block name="Java">
+
+  ```java
+    Amplify.DataStore.save(
+            item,
+            success -> Log.i("Tutorial", "Saved item: " + success.item.getName()),
+            error -> Log.e("Tutorial", "Could not save item to DataStore", error)
+    );
+  ```
+
+  </amplify-block>
+
+  <amplify-block name="Kotlin">
+
+  ```kotlin
+  Amplify.DataStore.save(
+          item,
+          { success -> Log.i("Tutorial", "Saved item: " + success.item.name) },
+          { error -> Log.e("Tutorial", "Could not save item to DataStore", error) }
+  )
+  ```
+
+  </amplify-block>
+  </amplify-block-switcher>
+
+1. Clear the logcat window by pressing the trashcan icon. Run the application. In logcat, you'll see an indication that the item was saved successfully:
```

Comment:
I wonder if it would be worth leaving the clear logcat step out since someone who doesn't know to do that already might be confused looking for this trashcan icon and thinking it's an important step and by default, Android Studio clears it anyway on a new run of the app.